client-app/src/Components/NewItemForm.tsx

```diff
@@ -0,0 +1,285 @@
+                    className = {`w-full px-3 py-2 rounded border ${errors[name] ? 'border-red-500' : 'border-gray-300'}`}
+                    title = "Upload 1-5 images in JPG or PNG format"         
+                />
+            ) : options ? ( // Only executes if there are options available
+            <select
+                id={name}
+                name={name}
+                value={formData[name]}
+                onChange={handleChange}
+                className={`w-full px-3 py-2 rounded border ${errors[name] ? 'border-red-500' : 'border-gray-300'}`}
+            >
+            <option value = "">Select {label}</option>
+            {options?.map(option => (
+                    <option key = {option.value} value = {option.value}>
+                        {option.label}
+                    </option>
+                ))}
+            </select>
+        ) : (
+            <input
+                type={type}
+                id={name}
+                name={name}
+                value={formData[name] as string}
+                onChange={handleChange}
+                className={`w-full px-3 py-2 rounded border ${errors[name] ? 'border-red-500' : 'border-gray-300'}`}
+                disabled = {name === 'currentStatus'}
+            />
+        )}
+        {errors[name] && (
+            <p className="text-red-500 text-sm mt-1">{errors[name]}</p>
+        )}
+        </div>
+    );
+
+    // HTML portion
+    return (
+        <div className="donor-form outer-container mx-auto p-10">
+            <h1 className="text-2xl font-bold heading-centered">
+                New Donated Item
+            </h1>
+            {errorMessage && <p className="error-message">{errorMessage}</p>}
+            {successMessage && (
+                <p className="success-message">{successMessage}</p>
+            )}
+            <form onSubmit={handleSubmit} className="form-grid">
+                {renderFormField('Item Type', 'itemType', 'text', true, itemTypeOptions)}
+                {renderFormField('Current Status', 'currentStatus')}
+                {renderFormField('Donor Email', 'donorEmail', 'text', true, donorEmailOptions)}
+                {renderFormField('Program', 'program', 'text', true, programOptions)}
+                {renderFormField('Image Upload', 'imageUpload', 'file')}
+                {renderFormField('Date Donated', 'dateDonated', 'date')}
+
+
+                <div className="form-field full-width button-container">
+                    <button type="submit" className="submit-button">
+                        Add Item
+                    </button>
+                    <button
+                        type="button"
+                        onClick={handleRefresh}
+                        className="refresh-button"
+                    >
+                        Refresh
+                    </button>
+                </div>
+            </form>
+        </div>
+    );
+};
+
+export default NewItemForm;
```

Comment:
This is same as donor form, this NewItemForm file should be imported in the home,  or else both donor form and donated item from should be integrated. Separate routes should be created.